src/ape/api/accounts.py

```diff
@@ -192,9 +268,23 @@ def append(self, account: AccountAPI):
         self.__setitem__(account.address, account)
 
     def __setitem__(self, address: AddressType, account: AccountAPI):
+        """
+        Add attribute to account.
+
+        Args:
+            address :class:`~ape.types.AddressType`
+            account :class:`~ape.accounts.AccountAPI`
+        """
         raise NotImplementedError("Must define this method to use `container.append(acct)`.")
 
     def remove(self, account: AccountAPI):
+        """
+        Delete an account by calling ``__delitem_``.
```

Comment:
```suggestion
        Delete an account.
```

How the account gets deleted is just an implantation detail, the user does not need to care about that.